providers/harvest/crawler.js

```diff
@@ -24,8 +24,30 @@ class CrawlingHarvester {
       method: 'POST',
       body,
       headers,
-      json: true
+      json: true,
+      resolveWithFullResponse: true
     })
+    .then(function (response) {
+        switch (response.statusCode) {
```

Comment:
@moranthomas  Is the status code returned to the caller? I can see the switch but what happens when e.g. 207 happen? is that handled and a message returned?

other than that, Given that the switch is quite similar in each case, would it make sense to move the logic in to its own function, etc. handleStatus or something like that, to avoid duplication?

Reply:
We tried to test the call from the website but we got into authentication issues - however the response codes are being returned so we should be good on this front.

I have also refactored the common code into it's own function